scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -416,6 +416,22 @@ global.trait_list = {
 		constitution : [2,2],
 		flavour_text:"Understands the ins and outs of defences both in building them and in taking them appart",
 		effect:"Bonus when commanding defences and extra boosts when leading a garrison",
+	},
+	"lobotomized" : {
+		wisdom : -50,
+		intelligence : -50,
+		charisma : -5,
+		constitution : 1, // Slight buff to health, as clear mind helps to stay healthy in some cases
+		technology : 1, // Also helps with some boring tasks, I think?
+		display_name : "Lobotomized",
+		flavour_text : "received treatment or damage, which resulted in portion of brain missing, akin to servitors",
+	},
+	"psychotic" : { // IT IS THE BANEBLAAADE!!! - Captain Diomedes, DoW 2 Retribution
+		wisdom : 10,
+		intelligence : -5,
+		constitution : -1, // Being bonkers can sometimes be damaging to one's health, I think..?
+		display_name : "Psychotic",
+		flavour_text : "whether due to experience or some other cause, acquired access to hidden knowledge...",
 	}
 }
```

Comment:
I am positive 100% about lobotomized trait, as I would have noticed at least 1 marine having negative or very low stats in wisdom and intelligence.
Psychotic, however, is not so easy to spot, hence I was not confident in my relatively quick glance at marines.

I am basing my knowledge on Gaussian randomization on trait list CM google calculator doc, which has a diagram there. Noticing that [10 (I think this is the mean);3 (1 sd, I think); "max" (I am not sure about this one)] gave negative or very low values at times, I prefer to avoid using Gaussian randomization, except for starting hireling and marine stats, but even then I tend to lower 1 sd, sometimes to just 1.

I am a bit skeptical using Gaussian randomization on most of the traits, as traits giving random stats seems odd to me, especially if it comes to true/false kind of situations - the person either knows or does not know.
However, I can see this being useful to experience traits, which are not so strictly defined as true/false, and varies from one person to another.
Lobotomite can be a good example for this - some person may suffer this injury in 1 section of the brain, other in a different one, hence the random stat affects from traits make sense.

Overall, how I understand the Gaussian randomization on traits, is that:
A Gaussian randomization runs on a trait, stats generated are saved, and applied on top of the marine stats.